api/applicationserver_integrations_alcsync.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "github.com/TheThingsIndustries/protoc-gen-go-json/annotations.proto";
+import "github.com/envoyproxy/protoc-gen-validate/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+package ttn.lorawan.v3;
+
+option go_package = "go.thethings.network/lorawan-stack/v3/pkg/ttnpb";
+
+enum ALCSyncCommandIdentifier {
+  option (thethings.json.enum) = { marshal_as_string: true, prefix: "CID" };
+
+  CID_PKG_VERSION = 0;
```

Comment:
So a pitty with the `enum` system in `protobuf` is that is subject to restrictions that appear in other languages, like C++.

In C++, enums have to be unique both within a single enum definition (that's quite reasonable), _and_ within a namespace (less obvious).

The namespace in this case is `ttn.lorawan.v3`, and we already use the `CID` notation for `MACCommandIdentifier`. If at some point we would have a MAC command with the same identifier, they would clash while generating C++ code.

The solution is to prefix the commands here (since we cannot break the `MACCommandIdentifier` API) with a prefix like `ALCSYNC`. I know it is verbose and doubles the naming, but it is what it is.